external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -181,40 +203,12 @@ export type TraceEvent =
         name: string,
         modInfo: ModuleInfo,
         localsTypes: string[],
-        paramValues: TraceValue[]
+        paramValues: RuntimeValueType[]
     }
     | { type: TraceEventKind.CloseFrame, id: number }
     | { type: TraceEventKind.Instruction, pc: number, kind: TraceInstructionKind }
     | { type: TraceEventKind.Effect, effect: EventEffect };
 
-/**
- * Kind of a location in the trace.
- */
-export enum TraceLocKind {
-    Local = 'Local'
-    // TODO: other location types
-}
-
-/**
- * Location in the trace.
- */
-export type TraceLocation =
-    | { type: TraceLocKind.Local, frameId: number, localIndex: number };
-
-/**
- * Kind of a value in the trace.
- */
-export enum TraceValKind {
```

Comment:
There is only one value kind in the runtime so no reason to maintain different kinds in trace after parsing (as they will never be used)